Pressure Interpolation

timothyas opened the pull request:
This adds code to interpolate from FV3's native vertical coordinate to pressure levels, following the code in https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere

danielabdi-noaa reviewed (approved):
@timothyas From a code-only point of view, it looks good to me but I can not comment much other than that. I suppose this is the code you used to try out diagnosing some variables instead of predicting them directly with GraphUFS. It is helpful that you have references to the FV3 code that contains the formulas used for conversion, thanks!

timothyas commented:
@danielabdi-noaa Thanks for looking this over. I moved one of the notebooks from the graphufs repo to here, and make it documentation for doing pressure interpolation. I also added and mentioned an example that computes geopotential.
